Form/Type/UpdateSelectFieldActionType.php

```diff
@@ -0,0 +1,82 @@
+<?php
+declare(strict_types=1);
+
+namespace MauticPlugin\LeuchtfeuerMultiselectHandlingBundle\Form\Type;
+
+use MauticPlugin\LeuchtfeuerMultiselectHandlingBundle\Form\Loader\LeadFieldChoiceLoader;
+use MauticPlugin\LeuchtfeuerMultiselectHandlingBundle\Form\Loader\LeadFieldValuesChoiceLoader;
+use MauticPlugin\LeuchtfeuerMultiselectHandlingBundle\Validator\UniqueMultiselectValues;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\NotBlank;
+
+class UpdateSelectFieldActionType extends AbstractType
+{
+    public const FIELD  = 'field';
+    public const ADD    = 'multiselect_add';
+    public const REMOVE = 'multiselect_remove';
+
+    public function __construct(private LeadFieldChoiceLoader $leadFieldChoiceLoader, private LeadFieldValuesChoiceLoader $leadFieldValuesChoiceLoader)
+    {
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $this->leadFieldChoiceLoader->setType($options['multiple']);
+        $this->leadFieldValuesChoiceLoader->setType($options['multiple']);
+        $fieldList = $this->leadFieldChoiceLoader->loadChoiceList()->getOriginalKeys();
+        if (null == $options['data'] && !empty($fieldList)) {
+            $flipFieldList = array_flip($fieldList);
+            $firstField    = reset($flipFieldList);
+            $this->leadFieldValuesChoiceLoader->setDefaultFieldId((int) $firstField);
+        } elseif (!empty($options['data']) && isset($options['data'][self::FIELD]) && $options['data'][self::FIELD] > 0) {
+            $this->leadFieldValuesChoiceLoader->setDefaultFieldId((int) $options['data'][self::FIELD]);
+        }
+        $builder->add(self::FIELD, ChoiceType::class, [
+            'label'         => 'mautic.plugin.multiselect_handling.field_action.managed_field',
+            'required'      => true,
+            'choice_loader' => $this->leadFieldChoiceLoader,
+            'constraints'   => [
+                new NotBlank(),
+            ],
+            'label_attr' => ['class' => 'control-label'],
+            'attr'       => [
+                'class'    => 'form-control',
+                'onchange' => 'Mautic.getOptionsFromField(this)',
+            ],
+            'multiple' => false,
+            'expanded' => false,
+        ])->add(self::ADD, ChoiceType::class, [
```

Comment:
Suggestion: this field could be renamed, since we are setting a value, not adding.